firefox/ScreenSharing/index.js

```diff
@@ -0,0 +1,35 @@
+var pageMod = require('sdk/page-mod');
+var prefsService = require('sdk/preferences/service');
+var allowDomainsPrefKey = 'media.getusermedia.screensharing.allowed_domains';
+var gDomains = ['*.example.com'];
+
+exports.main = function (options) {
+
+  if (options.loadReason === 'install' || 'enable') {
```

Comment:
I think you meant this:
```js
if (options.loadReason === 'install' || options.loadReason === 'enable')
```

Alternatively, you can test against an array of strings like this:
```js
if (['install', 'enable'].indexOf(options.loadReason) !== -1)
```